dropwizard-jackson/pom.xml

```diff
@@ -57,6 +57,10 @@
             <groupId>com.fasterxml.jackson.module</groupId>
             <artifactId>jackson-module-afterburner</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-blackbird</artifactId>
+        </dependency>
```

Comment:
Should we make these dependencies optional or rather rely on people to exclude the ones they're not using?

In either case, we'll have to detect the availability of each module in `io.dropwizard.jackson.Jackson`.

Reply:
I've seen a lot of success with the `optional` model for SDKs I've worked on. It also makes it more difficult for end users to wind up with a conflict :)

Thanks again for all the work on this!

Reply:
Agreed, I think going with the optional dependency would be the best way to go. Let's me see what would be the best way to detected Jackson modules in the classpath.

Reply:
The only way I've seen used is to use `Class.forName` on a fully-qualified class name. It's what's used in `DropwizardKafkaUtils` and `FileDescriptorRatioGauge`.

If there's a centralized place to put this logic (like `dropwizard-util`), that could simplify things.